src/components/Beacon/Beacon.js

```diff
@@ -0,0 +1,403 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Transition, animated } from 'react-spring'
+import { Helmet } from 'react-helmet'
+import {
+  Button,
+  IconClose,
+  SafeLink,
+  breakpoint,
+  springs,
+  theme,
+} from '@aragon/ui'
+import { GU } from '../../utils'
+import IconQuestion from './IconQuestion'
+import logo from './logo.png'
+
+const HELPSCOUT_BEACON = 'helpscout-beacon'
+const BEACON_EMBED =
+  '!function(e,t,n){function a(){var e=t.getElementsByTagName("script")[0],n=t.createElement("script");n.type="text/javascript",n.async=!0,n.src="https://beacon-v2.helpscout.net",e.parentNode.insertBefore(n,e)}if(e.Beacon=n=function(t,n,a){e.Beacon.readyQueue.push({method:t,options:n,data:a})},n.readyQueue=[],"complete"===t.readyState)return a();e.attachEvent?e.attachEvent("onload",a):e.addEventListener("load",a,!1)}(window,document,window.Beacon||function(){});'
+const BEACON_INIT =
+  "window.Beacon('init', '163e0284-762b-4e2d-b3b3-70a73a7e6c9f')"
```

Comment:
Is the second parameter our project identifier? 
What do you think about moving that to a constant and interpolating it?